.github/workflows/tests.yml

```diff
@@ -26,7 +26,11 @@ jobs:
         shell: bash -l {0}
         run: |
           mamba install --file requirements.txt pre-commit coverage coveralls mock pytest pytest-cov pip ${CONDA_PKGS} --yes --quiet
-          python -m pip install -e .
+          # get pyproject.toml out of the way so setuptools doesn't complain
+          mv pyproject.toml pyproject.toml.bak
+          python -m pip install -vv -e .
+          # restore so pre-commit's black get its config
+          mv pyproject.toml.bak pyproject.toml
```

Comment:
Given this just has a `black` setting in it, wonder if we should just move this into a `black` specific config file to avoid issues with other build tools

Reply:
Black _only_ supports pyproject.toml 😬 

https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#configuration-via-a-file

Reply:
`black` does take [a `--line-length` CLI argument]( https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html?highlight=--line-length#line-length ). Maybe we can just pass this to `black` directly?

Reply:
We could in CI, but people developing locally won't have that config setting and will reformat everything with the defaults.